sdk/src/types/block/signature/ed25519.rs

```diff
@@ -17,7 +17,7 @@ use packable::{
 use crate::types::block::{address::Ed25519Address, Error};
 
 /// An Ed25519 signature.
-#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
```

Comment:
Probably moving this type - if possible - is cheaper?

Reply:
What do you mean?

Reply:
This type is about 100 bytes in length, so I was wondering whether we should make it `Copy`.

Reply:
I'm not particularly concerned about it 🤷‍♀️

Reply:
Okay. Any more opinions from the team?

Reply:
It probably makes 0 difference